versioned_docs/version-3.0.0-beta/framework-guide/deploy-vite.md

```diff
@@ -0,0 +1,58 @@
+---
+id: deploy-vite
+title: How to Deploy Vite Apps on Spheron
+sidebar_label: Deploy Vite App
+slug: /framework-guide/deploy-vite
+---
+
+This guide will show you how to deploy a Vite project and get your domain up.
+
+[Vite](https://vitejs.dev/) aims to address these issues by leveraging new advancements in the ecosystem: the availability of native ES modules in the browser, and the rise of JavaScript tools written in compile-to-native languages.
+
+## Follow these steps to deploy your Vite app on Spheron.
+
+### Step 1: Put Your Vite Project Code in a Git Repo
+
+While this feature is not new to developers, the magic of Spheron is that every push to any branch in your GitHub, GitLab, or Bitbucket repo triggers a new build and deployment of your project.
+
+### Step 2: Deploy using the Spheron Platform UI.
```

Comment:
We need to have another section for vite for "Deploying on Decentralised Storage" just like react and vue. You need to tell the user what they need to change before they can deploy on spheron. Also here is the ref for it - https://mirror.xyz/dhaiwat.eth/NV--7dv8CO0NCcFCvRjDCxBe3VuxdB2_KggwFEfLGRc

Just checkout the "Prep for deployment to Arweave" part and convert it to our lang.

Reply:
@rekpero so this vite i deployed was VanillaJs (no framework) i guess i will see to how vite react works

Reply:
yup, checkout dhaiwat blog post on how to update setting to make it work with arweave and ipfs

Reply:
so @rekpero  i did a few findings and I pushed a new vite-react app (without the prep for deployment part) on both IPFS and Areweave and they work well (same app dhaiwat used).

I discovered that deploying to areweave and ipfs directly does not give their users options to configure the build options as we do hence the reason to...

I have deployed VUE and react vite app to confirm that we do not need the prep for the deployment part since users can config this options on our platform before deploying

Reply:
ok cool then, we are sorted then. merging this PR